pkg/v21/resource/appmigration/error_internal.go

```diff
@@ -0,0 +1,21 @@
+package appmigration
+
+import (
+	"strings"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	errorText = "the server could not find the requested resource (get chartconfigs.core.giantswarm.io)"
+)
+
+func isChartConfigNotInstalled(err error) bool {
```

Comment:
Is this correct here? The code is added but not used. There seem also to be some merge conflicts not resolved yet.

Reply:
Oh wow I need more coffee. No that shouldn't be committed. Thanks I'll get it sorted.

Reply:
Fixed.